### PR TITLE
remove bot account creation

### PR DESCRIPTION
**Pull request by bbodenmiller**
#### Summary
I don't see an option to set bot account anymore. Is the bot account now automatically created?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +64,6 @@ This plugin enables you to interact with jobs via slash commands in Mattermost.
     1. Go to the **System Console -> Plugins -> Jenkins**
     2. Set the Jenkins server URL along with the protocol. Example: http://jenkins.example.com, https://jenkins.example.com
     3. Save the settings
-3. Configure a bot account
-    1. Create a new Mattermost user, through the regular UI or the CLI with the username "Jenkins"
-    2. Go to the **System Console -> Plugins -> Jenkins** and select this user in the User setting
-    3. Save the settings
 4. Generate an at rest encryption key
```
Comment (hanzei) [inline README.md:67]: Would you please fix the numbering here and for the following items?
```suggestion
3. Generate an at rest encryption key
```

**Comment by bbodenmiller** [comment]
@hanzei updated. Changed to unnumbered as Markdown will render numbers.

**Comment by DHaussermann** [comment]
LGTM!

Thanks @bbodenmiller thanks for noticing this and keeping the info current.

**Comment by hanzei** [comment]
@aaronrothschild Heads up that I'm just going to merge this PR

**Pull request merged**